scripts/jenkins/dist-feed/dist-feed-manifest.yml

```diff
@@ -120,6 +120,12 @@ packages:
   dist-nilrt-grub_21.8_x64:
     export: "{{ (MNT_NIRVANA_PERFORCEEXPORTS + '/build/exports/ni/nilr/nilrt_onert_system_image/official/export/21.8') | latest_export }}"
     ipk_path: 'targets/linuxU/x64/gcc-4.7-oe/release/onert_system_image_ipk/dist-nilrt-grub_*.ipk'
+  dist-nilrt-grub_22.0_arm:
+    export: "{{ (MNT_NIRVANA_PERFORCEEXPORTS + '/build/exports/ni/nilr/nilrt_onert_system_image/prototype/export/22.0') | latest_export }}"
+    ipk_path: 'targets/linuxU/armv7-a/gcc-4.7-oe/release/onert_system_image_ipk/dist-nilrt-grub_*.ipk'
```

Comment:
I don't think this path exists; does it? We're not supporting ARM with this component anymore.

Reply:
Totally exists -- I'll drop the line though, so we don't pick it up and make it available.